modules/dop/component-protocol/components/issue-manage/issueTable/render.go

```diff
@@ -40,9 +40,16 @@ import (
 	"github.com/erda-project/erda/modules/openapi/hooks/posthandle"
 )
 
+type Multiple struct {
+	RenderType string        `json:"renderType,omitempty"`
+	Direction  string        `json:"direction,omitempty"`
+	Renders    []interface{} `json:"renders,omitempty"`
```

Comment:
I think the actual interface type is `Progress` only.

Reply:
done